src/components/dataset/data/sidebar/filter/CategoryFilter.jsx

```diff
@@ -33,7 +33,7 @@ export class CategoryFilter extends React.PureComponent {
           <Checkbox
             checked={checked}
             onChange={this.handleChange}
-            value={name}
+            value={name ?? "null"}
```

Comment:
Does it make sense to have this be the italic `(empty)` like we show in the table?

Reply:
I agree, I think "Empty" or "None" would be clearer!

Reply:
Great! I will cover this in my next UI PR!